lib/t/accounts/profile.ex

```diff
@@ -5,27 +5,30 @@ defmodule T.Accounts.Profile do
   @primary_key false
   @foreign_key_type Ecto.Bigflake.UUID
   schema "profiles" do
-    belongs_to :user, T.Accounts.User, primary_key: true
+    belongs_to(:user, T.Accounts.User, primary_key: true)
 
-    field :story, {:array, :map}
-    field :location, Geo.PostGIS.Geometry
+    field(:story, {:array, :map})
+    field(:location, Geo.PostGIS.Geometry)
```

Comment:
...here it results in unnecessary diffs. Many other files would be changed as well.